hack/vendor.sh

```diff
@@ -57,6 +57,8 @@ clone git github.com/Sirupsen/logrus v0.7.2
 
 clone git github.com/go-fsnotify/fsnotify v1.0.4
 
+clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
```

Comment:
Since it's only in integration-cli, would it be better to put it in the
Dockerfile, or does it make sense for us to vendor this code too?

Interested in getting more maintainer opinions besides just mine and yours
without tainting the discussion myself too much, since I'm unsure which
would be better myself. :smile: